### PR TITLE
Fix URL of stats reviewer guide

### PR DESCRIPTION
**Pull request by noamross**
Current URL is a 404

### CODE REVIEW

**Comment by mpadge** [comment]
Thanks Noam!

File: app/responders/ropensci/reviewers_due_date_responder.rb [hunk 1]
```diff
@@ -23,7 +23,6 @@ def process_message(message)
       add_to_or_remove_from = [add_or_remove, to_or_from].join(" ")
 
       if ["add to reviewers", "add as reviewer", "assign as reviewer", "assign to reviewers"].include?(add_to_or_remove_from)
-        add_reviewer
```
Comment (maelle) [inline app/responders/ropensci/reviewers_due_date_responder.rb:26]: why this?

Comment (noamross) [reply app/responders/ropensci/reviewers_due_date_responder.rb:26]: I have no idea how I did that. I was wondering how changing a URL would break tests.

**Comment by maelle** [comment]
closing in favor of #85

**Pull request closed without merging**